### PR TITLE
shrksl: Added CALayer Extension

### PR DESCRIPTION
**Pull request by kusalshrestha**

### CODE REVIEW

File: Extensions/CALayer+Extensions.swift [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  public class func animateWithDuration(duration: TimeInterval, animation: () -> Void, completion: (() -> Void)?) {
+    CATransaction.begin()
+    CATransaction.setAnimationDuration(duration)
+    CATransaction.setCompletionBlock(completion)
+    animation()
+    CATransaction.commit()
+  }
+
+}
+
```
Comment (sanjaymhj) [inline Extensions/CALayer+Extensions.swift]: extra line at end :)

**Comment by sanjaymhj** [comment]
Also update README.md

File: Extensions/CALayer+Extensions.swift (hunk 1 above)
Comment (kusalshrestha) [reply Extensions/CALayer+Extensions.swift]: Why are we adding an extra line at the EOF?

Comment (sanjaymhj) [reply Extensions/CALayer+Extensions.swift]: My guess is there might be 2 line at the end of file.

Comment (kusalshrestha) [reply Extensions/CALayer+Extensions.swift]: Nope.. But I guess your file UIColor+extension dont have any extra line at the end  :D lol

Comment (sanjaymhj) [reply Extensions/CALayer+Extensions.swift]: There is github just doesnot show one line :)

**Comment by sanjaymhj** [comment]
👍

**Review by sanjaymhj — COMMENTED** [review]
👍

**Review by sanjaymhj — APPROVED** [review]

**Pull request merged**